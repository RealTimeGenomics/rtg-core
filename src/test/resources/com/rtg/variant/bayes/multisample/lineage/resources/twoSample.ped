lineage original 0 0 2 0
lineage descended 0 original 2 0
