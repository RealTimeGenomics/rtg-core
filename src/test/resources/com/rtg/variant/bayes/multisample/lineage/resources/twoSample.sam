@HD	VN:1.3	SO:coordinate
@SQ	SN:ref	LN:50
@RG	ID:original	PL:ILLUMINA	LB:original	PI:300	DS:Simulated dataset	DT:2012-09-25T12:00:00+1200	SM:original
@RG	ID:descended	PL:ILLUMINA	LB:lb_descended	PI:300	DS:Simulated dataset	DT:2012-09-25T12:00:00+1200	SM:descended
@PG	ID:rtg	PN:rtg	VN:vPOST-2.6 build <not found> (<not found>)	CL:map --repeat-freq=100% --read-names --template template.sdf --tempdir=mytemp --legacy-cigars -m 150 -M 450 --input reads_dad.sdf --output map_dad --sex=male --sam-rg /dev/fd/63
@CO	RUN-ID:f2e8d04c-81ec-42f5-b92c-fef7d4af68c1
@CO	READ-SDF-ID:40fc7b93-8ff6-4b7b-86d0-dad67dc4786c
@CO	MAPPING-GENDER:MALE
@CO	TEMPLATE-SDF-ID:00000000-0000-0000-0000-00000000002a
original0	99	ref	1	55	50M	*	0	0	ATGATTCCGCGCGTGCTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	55555555555555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
descended0	99	ref	1	55	50M	*	0	0	ATGATTCCGCGCGTGCTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	55555555555555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:descended	IH:i:1	NH:i:1
original1	99	ref	2	55	49M	*	0	0	TGATTCCCCGCGTGCTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	5555555555555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
descended1	99	ref	2	55	49M	*	0	0	TGATTCCCCGCGTGCTTACTCGTTCTGGCGGTGTAAGACAAGGCTCTAT	5555555555555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:descended	IH:i:1	NH:i:1
original2	99	ref	3	55	48M	*	0	0	GATTCCGCGCGTGCTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	555555555555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
descended2	99	ref	3	55	48M	*	0	0	GATTCCGCGCGTGCTTACTCGTTCTGGCGGTGTAAGACAAGGCTCTAT	555555555555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:descended	IH:i:1	NH:i:1
original3	99	ref	4	55	47M	*	0	0	ATTCCCCGCGTGCTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	55555555555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
descended3	99	ref	4	55	47M	*	0	0	ATTCCCCGCGTGCTTACTCGTTCTGGCGGTGTAAGACAAGGCTCTAT	55555555555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:descended	IH:i:1	NH:i:1
original4	99	ref	5	55	46M	*	0	0	TTCCGCGCGTGCTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	5555555555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
descended4	99	ref	5	55	46M	*	0	0	TTCCGCGCGTGCTTACTCGTTCTGGCGGTGTAAGACAAGGCTCTAT	5555555555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:descended	IH:i:1	NH:i:1
original5	99	ref	6	55	45M	*	0	0	TCCCCGCGTGCTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	555555555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
descended5	99	ref	6	55	45M	*	0	0	TCCCCGCGTGCTTACTCGTTCTGGCGGTGTAAGACAAGGCTCTAT	555555555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:descended	IH:i:1	NH:i:1
original6	99	ref	7	55	44M	*	0	0	CCCCGCGTGCTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	55555555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
original7	99	ref	8	55	43M	*	0	0	CCCGCGTGCTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	5555555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
original8	99	ref	9	55	42M	*	0	0	CCGCGTGCTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	555555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
original9	99	ref	10	55	41M	*	0	0	CGCGTGCTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	55555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
original10	99	ref	11	55	40M	*	0	0	GCGTGCTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	5555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
original11	99	ref	12	55	39M	*	0	0	CGTGCTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	555555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
original12	99	ref	13	55	38M	*	0	0	GTGCTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	55555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
original13	99	ref	14	55	37M	*	0	0	TGCTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	5555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
original14	99	ref	15	55	36M	*	0	0	GCTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	555555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
original15	99	ref	16	55	35M	*	0	0	CTTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	55555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
original16	99	ref	17	55	34M	*	0	0	TTACTCGTTCTGGCGGTGAAAGACAAGGCTCTAT	5555555555555555555555555555555555	AS:i:2	NM:i:2	MD:Z:48G37C13	XA:i:4	RG:Z:original	IH:i:1	NH:i:1
