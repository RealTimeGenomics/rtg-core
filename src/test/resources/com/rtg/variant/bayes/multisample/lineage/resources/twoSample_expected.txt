ref	9	.	G	C	.	PASS	DP=15	GT:DP:RE:AR:RQ:GQ:DN:DNP:ABP:SBP:RPB:PPB:AQ:PUR:RS:AD:GL	0/1:6:0.060:0.000:138.6:105:N:134:0.00:6.51:13.03:0.00:59.996,59.996:0.00:C,3,0.030,G,3,0.030:3,3:-13.86,0.00,-10.54	0/1:9:0.090:0.000:139.7:105:.:.:2.17:13.03:19.54:0.00:59.996,119.992:0.00:C,6,0.060,G,3,0.030:3,6:-13.97,0.00,-10.54
ref	35	.	A	T	.	PASS	DP=23	GT:DP:RE:AR:RQ:GQ:DN:DNP:ABP:SBP:RPB:PPB:AQ:PUR:RS:AD:GL:OCOC:DCOC:OCOF:DCOF	0/1:6:0.060:0.000:20.8:21:Y:7:5.79:10.86:13.03:0.00:19.999,99.993:0.00:A,1,0.010,T,5,0.050:1,5:-2.08,0.00,-10.24:0:0:0.000:0.000	0/0:17:0.170:0.000:1.0:7:.:.:0.00:36.92:36.92:0.00:339.977,0.000:0.00:A,17,0.170:17,0:-0.10,-0.67,-38.48
